docs/tapiro/installation.md

```diff
@@ -0,0 +1,100 @@
+---
+id: installation
+title: Installation
+---
+
+`tapiro` can be installed as an sbt-plugin.
+
+`sbt-tapiro` is an Sbt plugin that uses `tapiro` to generate http/json routes parsing scala traits definitions.
+
+## Installation
+
+To start using `sbt-tapiro` simply add this line in `project/plugins.sbt`
+
+```scala
+addSbtPlugin("io.buildo" %% "sbt-tapiro" % "@SBT_TAPIRO_STABLE_VERSION@")
+```
+
+### Snapshot releases
+
+We publish a snapshot version on every merge on master.
+
+The latest snapshot version is `@SBT_BUILDO_SNAPSHOT_VERSION@` and you can use
+it to try the latest unreleased features. For example:
+
+```scala
+addSbtPlugin("io.buildo" %% "sbt-tapiro" % "@SBT_TAPIRO_SNAPSHOT_VERSION@")
+resolvers += Resolver.sonatypeRepo("snapshots")
+```
+
+## Plugin
+
+To use the code generator, you need to add this to your `build.sbt`.
+
+```scala
+lazy val application = project
+  .settings(
+    libraryDependencies ++= applicationDependencies ++ tapiroDependencies,
+    tapiro / tapiroRoutesPaths := List("[path to routes]"),
+    tapiro / tapiroModelsPaths := List("[path to models]"),
+    tapiro / tapiroOutputPath := "[path to endpoints]",
+    tapiro / tapiroEndpointsPackages := List("[package]", "[subpackage]"),
+    tapiro / tapiroServer := Server.AkkaHttp, //or Server.Http4s
+  )
+  .enablePlugins(SbtTapiro)
+```
+
+You can now run it with `sbt application/tapiro`.
+
+In order to use `tapiro` you also need to add the macro paradise plugin (for
```

Comment:
Why? Usiamo macro annotations in tapiro?

Reply:
In effetti forse è un refuso